other-builds/ndkbuild/bitmap-plasma/app/build.gradle

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'com.android.application'
+
+// pointing to cmake's source code for the same project
+def REMOTE_PROJ_ROOT = '../../../../' + rootProject.getName() + '/' +
+                        project.getName() + '/src/main'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        applicationId "com.example.plasma"
+        minSdkVersion 8
+        targetSdkVersion 8
+        ndkConfig.abiFilters 'armeabi'
```

Comment:
yes will update for that: ndk { abiFilters.. } works, will trying to save some space inside gradle file